addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -599,4 +596,5 @@ Object.assign(AttachmentViewer, {
     template: 'mail.AttachmentViewer',
 });
 
+registerMessagingComponent(AttachmentViewer);
 QWeb.registerComponent('AttachmentViewer', AttachmentViewer);
```

Comment:
also probably not needed?

Reply:
I was wondering what those global registrations were for, although honestly I'm still kind of confused at how either of those components is instantiated so I didn't want to remove them without understanding their lifecycle first. I'll try to remove them and see if anything breaks